setup.py

```diff
@@ -20,6 +20,11 @@
 from shutil import copy
 
 try:
+    import setuptools
+except ImportError:
+    pass  # setuptools allows for "develop" and entry_points, but it's not essential
```

Comment:
Sorry my question is: should we do this unconditionally all the time, or restrict to only when we know we need it (i.e. installs on windows)